src/beautifiers/sqlformat.coffee

```diff
@@ -17,7 +17,7 @@ module.exports = class Sqlformat extends Beautifier
   beautify: (text, language, options) ->
     @run("sqlformat", [
       @tempFile("input", text)
-      "--reindent=#{options.reindent}" if options.reindent?
+      "--reindent" if (options.reindent? && options.reindent == true)
```

Comment:
I think this should be:

```coffeescript
"--reindent" if options.reindent is true
```

Which is the following in JavaScript:

```javascript
if (options.reindent === true) {
  "--reindent";
}
```

Reply:
What about the situation that options.reindent is null or undefined? Or it is impossible to meet that situation.

Reply:
It could happen and then it would not add `--reindent` option at all. Is this the desired behaviour?